sdk/include/opentelemetry/sdk/metrics/state/attributes_hashmap.h

```diff
@@ -66,6 +78,11 @@ class AttributesHashMap
       return it->second.second.get();
     }
 
+    if (hash_map_.size() >= attributes_limit_)
```

Comment:
Per the spec:

"
The SDK MUST create an Aggregator with the overflow attribute set prior to reaching the cardinality limit and use it to aggregate events for which the correct Aggregator could not be created. The maximum number of distinct, non-overflow attributes is one less than the limit, as a result.
"

So, with a limit of 10, there should be room for 9 + 1 aggregations.

The unit tests show a size of 11 instead of 10.

How about:

```
if (hash_map_.size() + 1 >= attributes_limit_)
```